code/_onclick/telekinesis.dm

```diff
@@ -109,7 +109,7 @@
 	if(!host || host != user)
 		qdel(src)
 		return
-	if(!host.dna?.GetSEState(GLOB.teleblock))
+	if(!host.dna && !HAS_TRAIT(host, TRAIT_TELEKINESIS))
```

Comment:
Even if we'll be keeping the `if(X.dna` checks that I mentioned in my other review, I just realized this is backwards as well. Now this will only pass if they suddenly have a null `dna` var, which is probably not what we want.
```suggestion
	if(host.dna && !HAS_TRAIT(host, TRAIT_TELEKINESIS))
```